datasources/sources/overpass.rb

```diff
@@ -86,22 +86,22 @@ def map_geometry(feat)
       }
     end
 
-    if feat['type'] == 'relation'
-      linestrings = feat['members'].select{ |m|
-        m['type'] == 'way'
-      }.collect{ |m|
-        m['geometry']
-      }.compact.collect { |g|
-        g.collect{ |g| [g['lon'], g['lat']] }
-      }
-
-      {
-        type: 'MultiLineString',
-        coordinates: linestrings
-      }
+    return if feat['geometry'].nil?
+
+    if feat['geometry'].is_a?(Hash) && feat['geometry']['type'] == 'GeometryCollection'
```

Comment:
It not just to check is not nil ? But you already check it before. I think condition could be removed?

```suggestion
    if feat['geometry']['type'] == 'GeometryCollection'
```